client/errors.go

```diff
@@ -47,6 +47,7 @@ var (
 	ErrMaxTxnRetries        = errors.New(errMaxTxnRetries)
 	ErrRelationOneSided     = errors.New(errRelationOneSided)
 	ErrCollectionNotFound   = errors.New(errCollectionNotFound)
+	ErrUnknownCRDT          = errors.New("unknown crdt")
```

Comment:
suggestion: It might be worth including the unknowns crdt value in the error.

EDIT: I see that this is not a new error, and very happy for you to shrug this suggestion off as out of scope for now.

Reply:
I like that suggestion. I'll do the change 👍.

Reply:
✅